src/solver.jl

```diff
@@ -47,3 +47,30 @@ is_solved(opt_model::J.Model) =
     J.termination_status(opt_model) in [J.MOI.OPTIMAL, J.MOI.LOCALLY_SOLVED]
 
 export is_solved
+
+"""
+    Minimal
+
+Objective sense for finding the minimal value of the objective.
+
+Same as `JuMP.MIN_SENSE`.
+"""
+const Minimal = J.MIN_SENSE
+
+"""
+    Maximal
+
+Objective sense for finding the maximal value of the objective.
+
+Same as `JuMP.MAX_SENSE`.
+"""
+const Maximal = J.MAX_SENSE
+
+"""
+    Maximal
+
+Objective sense for finding the any feasible value of the objective.
+
+Same as `JuMP.FEASIBILITY_SENSE`.
+"""
```

Comment:
My concern here is that it is just more stuff to remember. Can we not just re-export the JuMP constants?

Reply:
one can use the jump constants as well. I just don't want to 1] cause identifier collision 2] have folks to import jump all over again everytime they want to do something